docs/source/installation/guide_quick/index.rst

```diff
@@ -0,0 +1,83 @@
+.. _installation:guide-quick:
+
+========================
+Quick installation guide
+========================
+
+.. attention::
+
+    AiiDA requires a recent version of Python.
+    Please refer to the `Python Package Index <https://pypi.org/project/aiida-core/>`_ for the minimum required version.
+
+First, install the ``aiida-core`` Python package:
+
+.. code-block:: console
+
+    pip install aiida-core
+
+Next, set up a profile where all data is stored:
+
+.. code-block:: console
+
+    verdi presto
+
+Verify that the installation was successful:
+
+.. code-block:: console
+
+    verdi status
+
+.. code-block:: console
+ ✔ version:     AiiDA v2.5.1.post0
+ ✔ config:      /path/.aiida
+ ✔ profile:     presto
+ ✔ storage: SqliteDosStorage[/path/.aiida/repository/sqlite_dos_8ee66fde9bd34e7c8fc8e0e29b20c429]: open,
+ ⏺ broker:      No broker defined for this profile: certain functionality not available.
+ ⏺ daemon:      No broker defined for this profile: daemon is not available.
+If all lines show green checkmarks, the installation was successful and you are good to go.
+
+.. admonition:: What next?
+    :class: hint
+
+    If you are a new user, we recommend to start with the :ref:`basic tutorial <tutorial:basic>`.
+    Alternatively, check out the :ref:`next steps guide <tutorial:next-steps>`.
+
+.. warning::
+
+    Not all AiiDA functionality is supported by the quick installation.
+    Please refer to the :ref:`section below <installation:guide-quick:limitations>` for more information.
+
+
+.. _installation:guide-quick:limitations:
+
+Quick install limitations
+=========================
+
+Functionality
+-------------
+
+Part of AiiDA's functionality requires a `message broker <https://en.wikipedia.org/wiki/Message_broker>`_, with the default implementation using `RabbitMQ <https://www.rabbitmq.com/>`_.
+The message broker is used to allow communication with the :ref:`daemon <topics:daemon>`.
+Since RabbitMQ is a separate service and is not always trivial to install, the quick installation guide sets up a profile that does not require it.
+As a result, the daemon cannot be started and processes cannot be submitted to it but can only be run locally.
+
+.. note::
+    The ``verdi presto`` command automatically checks if RabbitMQ is running on the localhost.
+    If it can successfully connect, it configures the profile with the message broker and therefore the daemon functionality will be available.
```

Comment:
The sentence you added is incorrect though. You don't need to have a daemon to run on remote computers. So RabbitMQ is not required to run calculations on remote computers

Reply:
I thought it's not possible to submit jobs without daemon running, maybe I'm wrong ?
But I agree it's better to stress that you can still run on remote computer but cannot submit.

Reply:
> I thought it's not possible to submit jobs without daemon running

What is probably confusing you is the usage of "submit" in AiiDA lingo. We distinguish between _running_ and _submitting_ processes. The former performs the process in a local runner, whereas the latter has the process executed by a daemon runner. This is where RabbitMQ is necessary because that is how we can communicate with daemon runners. It has nothing to do with "submitting" to a scheduler on a remote computer.

Reply:
ok, I see, thanks.
Then I would really change the last sentence in functionality paragraph. As it's a technical thing, it conveys the confusing message to beginners like me. Those who want easy installation might not be familiar with AiiDA lingo:

"As a result, the daemon cannot be started and processes cannot be submitted to it but can only be run locally."

Reply:
Do you have any suggestions? Not sure how better to explain it. I guess I could add the literal consequence:
```
That is to say, it is only possible to use :meth:`aiida.engine.launch.run` to launch a process and :meth:`aiida.engine.launch.submit` is not available.
```
But that is even more technical and may confuse new users even more. I really think the simplest is just saying "the daemon functionality is not available" (which means starting the daemon and submitting processes to it). How else would you phrase/explain it?

Reply:
I would say it explicitly in real application. It's a important piece of information that submitting to remote computer is functional.

Maybe this ?:

++"As a result, the daemon functionality is not available. Note, although easy instillation is fully functional with submitting jobs to a remote computer, some features [like this and that] are limitted, if you plan to heavily load AiiDA with synchronous tasks we recommend install RabbitMQ :ref:`guide to install RabbitMQ <installation:guide-complete:create-profile:rabbitmq>` to make daemon functional."

--"As a result, the daemon cannot be started and processes cannot be submitted to it but can only be run locally."

Reply:
Ok, fair enough. I guess I just don't really understand why you even thought that running on remote computers was not supported without the daemon. Nowhere was this particular feature of AiiDA mentioned. So it seems weird to me to single this one out as explicitly stating that it _is_ still supported without a broker. Should we also mention that caching is still supported? Or workflows, or any other feature?

Reply:
Well, honestly this was a question raised during Marvel review a few days ago. Some body asked if with the new `verdi presto` one can still submit jobs, we didn't know the answer, and although many of us, were present, nobody corrected it... that made me think maybe it is like that.. perhaps AiiDA lingo confusion also playing a part..

Apart from that, to be honest with you, I don't understand at all what RabbitMQ does exactly in AiiDA unfortunately (apart from being a message broker, etc....), I haven't explored that part of AiiDA yet.

Reply:
In the risk of creating even more confusion… I think the misunderstanding here comes from the fact that you are talking about two different things. One is *executing tasks* on **remote computers**, which shouldn't have anything to do with RMQ / `verdi presto`. While the other thing is *running* (that is, blocking) vs. *submitting* to the *AiiDA deamon*, where using a non-RMQ profile doesn't allow doing the latter (irrespective if the task should be executed locally or remotely). Note the distinction between submitting to the **AiiDA daemon** (-> requires RMQ) vs. submitting to a scheduler, e.g. SLURM, irrespective if on a local or remote machine (-> doesn't require RMQ). At least that's the way I understand it, feel free to correct me if I'm wrong.